### PR TITLE
Remove vgs-client header from request

### PR DESCRIPTION
**Pull request by aslepakurov**
## Description of changes in release / Impact of release:
Removed `vgs-client` header for request, coming from vgs-collect 

### Is this a breaking change?
  - [ ] Yes
  - [X] No

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ dist: clean
 	 app.py
 
 test_dist:
-	pytest satellite/tests -m "dist"
+	pytest satellite/tests -m "dist" -v
```
Comment (rozenbom) [inline Makefile:32]: Usually, you don't want to clutter CI output and if something fails you check it locally. If you really want to see verbose test output on CI you need `-vv`-option.

Comment (rozenbom) [reply Makefile:32]: Also if you 100% believe in your changes just run pytest with `--snapshot-update` option locally to update snapshots.

Comment (aslepakurov) [reply Makefile:32]: accidently commited local debugging session, my bad

File: satellite/proxy/master.py [hunk 2]
```diff
@@ -32,6 +32,10 @@ def running(self):
         signal('sat_proxy_started').send(self)
 
     def request(self, flow: HTTPFlow):
+        # Remove "vgs-client" header, that is sent by VGS-Collect.
+        # This copies logic from proxy
+        if 'vgs-client' in flow.request.headers:
+            flow.request.headers.pop('vgs-client')
```
Comment (rozenbom) [inline satellite/proxy/master.py:38]:
```suggestion
        flow.request.headers.pop('vgs-client', None)    
```

**Review by rozenbom — APPROVED** [review]

**Pull request merged**